packages/ncids-js/src/index.ts

```diff
@@ -7,6 +7,9 @@ export { NCISubscribe } from './components/nci-subscribe/nci-subscribe.component
 import { NCIExtendedHeaderWithMegaMenu } from './components/nci-header/index';
 export { NCIExtendedHeaderWithMegaMenu };
 
+import { NCIExtendedHeaderWithMegaMenuAutoInit } from './components/nci-header/index';
```

Comment:
This should not be in the index. If you:
```
import { NCIExtendedHeaderWithMegaMenu } from './components/nci-header';
```
In something like CGDP, we will be bundling the index.js module. Which means we should also be getting the `NCIExtendedHeaderWithMegaMenuAutoInit` pulled along.

I was thinking that others would do something like:
```
import { NCIExtendedHeaderWithMegaMenu } from '@nciocpl/ncids-js/nci-header/auto-init';
```